csharp/README.md

```diff
@@ -26,20 +26,17 @@ in the [Ice manual][1].
 Refer to the [C++ demos)(../cpp) for more examples that use the Ice services
 (Glacier2, IceGrid, IceStorm).
 
-## Building and running the Demos with .NET on Windows, Linux, and macOS
+## Building and running the Demos
 
 ### Build Requirements
 
 In order to build Ice for .NET sample programs, you need all of the following:
 
-- the [.NET SDK][5] version 6.0 or later
-- Slice to C# compiler, for Windows platform the Slice to C# compiler is included
-  with the `zeroc.ice.net` NuGet package installed during build, for other platforms
-  refer to the [Release Notes][6].
+- the [.NET SDK][3] version 8.0 or later
 
 You can build from the command-line or with Visual Studio 2022 for Windows.
 
-### Building the demos using NuGet packages
+### Building the demos
```

Comment:
Should we mention that a local source build is currently required?

Reply:
Yes we can do that.